python-server/helper.py

```diff
@@ -0,0 +1,30 @@
+import cv2
+import json
+
+# Saves an array as a JSON in a file
+
+
```

Comment:
Maybe remove the two line before the method to put the comments closer to the method

Reply:
Thanks! Fixed them in the new commit